frontend/src/components/common/form/AutoCompleteText.tsx

```diff
@@ -105,7 +105,7 @@ export const AutoCompleteText: React.FC<IAutoCompleteProps> = ({
         <Form.Control
           autoComplete={autoSetting}
           name={field}
-          value={text}
+          value={text ? text : value}
```

Comment:
I think you can just use `text ?? value`

Reply:
oooh good to know! thanks